uberspace/roles/uberspace_ephios/tasks/postgres.yml

```diff
@@ -0,0 +1,3 @@
+- name: set database url
+  set_fact:
+    database_url: "psql://ephios@localhost:5432/ephios"
```

Comment:
it isn't. I did the setup manually beforehand and assumed a working postgres as a prerequisite just as we assume a working mysql installation

Reply:
We assume a somewhat clean uberspace. That includes a MySQL but not postgres. I think the *right* thing to do would be to include the postgres setup in Ansible, but I'd accept postponing that.

Reply:
but lets not write that role for ourselves. Jim has something nice prepared (https://git.2martens.de/2martens/uberspace-ansible/src/branch/master/roles/postgres), although it seems a bit outdated because this role compiles postgres manually. https://githubmemory.com/repo/friesenkiwi/ansible-collection-uberspace also announced a role based on the UberLab guide

Reply:
moved to #7